apps/akasha/src/extensions/entry-remove-modal.tsx

```diff
@@ -0,0 +1,191 @@
+import singleSpaReact from 'single-spa-react';
+import * as React from 'react';
+import ReactDOM from 'react-dom';
+import { RootComponentProps } from '@akashaproject/ui-awf-typings';
+import DS from '@akashaproject/design-system';
+import { useErrors, useLoginState, usePosts } from '@akashaproject/ui-awf-hooks';
+import i18n from 'i18next';
+import { I18nextProvider, initReactI18next, useTranslation } from 'react-i18next';
+import LanguageDetector from 'i18next-browser-languagedetector';
+import Backend from 'i18next-chained-backend';
+import Fetch from 'i18next-fetch-backend';
+import LocalStorageBackend from 'i18next-localstorage-backend';
+import { QueryClientProvider, QueryClient } from 'react-query';
+import getSDK from '@akashaproject/awf-sdk';
+import { events } from '@akashaproject/awf-sdk/typings';
+import { filter } from 'rxjs/operators';
+
+const { ThemeSelector, lightTheme, darkTheme, ConfirmationModal } = DS;
+
+const queryClient = new QueryClient();
```

Comment:
you don't need this anymore if you're not using hooks from `use-posts.new`

Reply:
you can safely remove it from package.json and the QueryClient provider

Reply:
in the future it will be migrated to react query, so it still needs it, but you just need to wrap the ModalWrapper with the provider-hoc.tsx withProviders from hooks, and that gives QueryClientProvider and ThemeSelector